tests/test.c

```diff
@@ -190,6 +190,7 @@ struct Test tests[] = {
     TEST_CASE_EXPECTED(test_atom_to_list, 1),
     TEST_CASE(test_display),
     TEST_CASE(test_integer_to_list),
+    TEST_CASE(test_port_to_list),
```

Comment:
The file wasn't added in this PR

Reply:
Yes, thanks, I already realized I added that to the wrong branch.